src/main/java/dtu/scheduler/SchedulingApp.java

```diff
@@ -39,7 +39,16 @@ public double getWeeklyRegisteredHours() {
 		return currentUser.getWeeklyRegisteredHours();
 	}
 
+	public double getHoursRegisteredOnActivity(Activity activity) throws Exception {
+		TimeRegistration registration = currentUser.getTimeRegistrationByActivity(activity);
+		return registration.getHours();
+	}
+
```

Comment:
This might be wrong. You only check current user, what if multiple users have have registered hours.

Reply:
Assuming that the schedulingApp only handles one user at a time, this gets the hours registered by the current user. Same as "registerHours" registering hours for the current user. Should probably change the name, but I imagine this is the kind of stuff to get changed in the "check hours" use case.